### DISCUSSION TITLE
FC 新版本发布0.1.30

### DISCUSSION THREAD

**Discussion by heimanba** [devsapp/fc#517] [Show and tell]
### s local 本地调试能力

- [x]  #382

- [ ]  #381

- [ ]  #509

- [x]  #486

- [x]  #203
- [ ]  #415
- [ ] #281
- [ ] #227

**Comment by wss-git** [comment]
### 功能
- [x]  plan 输出样式不好看： https://github.com/devsapp/fc/issues/512
- [x]  function environmentVariables 输出优化(deploy 输出优化)：https://github.com/devsapp/fc/issues/511
- [x]  dev 版本问题优化：https://github.com/devsapp/fc/issues/507
- [x]  Deploy时的线上资源感知能力，提示的有些误导（plan）：https://github.com/devsapp/fc/issues/503
- [x]  s build help 提示信息有错误：https://github.com/devsapp/fc/issues/500
- [x]  s init 模板共享同一个 FC，每次初始化一个别的模板，都会触发 s 检查已有 service 存在，需要选择 local 还是 remote 配置（plan）：https://github.com/devsapp/fc/issues/493
- [x]  本地调试时监听端口和 s.yaml 中的 caPort 不一致时，实例hang住，无法终止: https://github.com/devsapp/fc/issues/486  https://github.com/devsapp/fc/issues/203
- [ ] 使用 remove 的时候，返回结果结构不合理: https://github.com/devsapp/fc/issues/482  【？？？？】
- [x] 关于deploy diff逻辑的判断( plan 的部分): https://github.com/devsapp/fc/issues/477
- [ ] deploy 和 sync 字段不一致( plan 的部分): https://github.com/devsapp/fc/issues/476
- [ ] 使用 custom container 导致的非预期 loca 和 remote 状态不一致( plan 的部分)：https://github.com/devsapp/fc/issues/484
- [x] s logs 帮助命令中，--type 参数的含义解释不清晰：https://github.com/devsapp/fc/issues/427
- [x] -t 模式下使用 --keyword 参数过滤日志，行为不符合预期：https://github.com/devsapp/fc/issues/426
- [x] s stress clean 提醒一个error：https://github.com/devsapp/fc/issues/418
- [x] s deploy 体验优化：https://github.com/devsapp/fc/issues/410
- [x] logConfig 为 auto 的时候， 出现 no access：https://github.com/devsapp/fc/issues/406
- [x] 不能执行 s nas command ls -lh /mnt/auto 指令：https://github.com/devsapp/fc/issues/394
- [x] [建议] 针对 http trigger 函数， s invoke 方法有点复杂：https://github.com/devsapp/fc/issues/274
- [x] custom domain 部署逻辑与 service 等资源部署逻辑保持一致 ：https://github.com/devsapp/fc/issues/115
- [x] 支持有状态异步调用：https://github.com/devsapp/fc/issues/54
- [x] s remove 能能力：https://github.com/devsapp/fc/issues/38
- [x] 调用函数的时候，携带请求头: https://github.com/devsapp/fc/issues/460 (MR: https://github.com/devsapp/fc-remote-invoke/pull/19)
- [x] 删除结束之后输出： https://github.com/devsapp/fc/issues/482
- [x] 部署指令增加 --dry-run 参数（说的应该就是 plan 命令）：https://github.com/devsapp/fc/issues/177

### 文档
- s plan能力：https://github.com/devsapp/fc/issues/451
- leyer：https://github.com/devsapp/fc/issues/450
- local：https://github.com/devsapp/fc/issues/449
- logs：https://github.com/devsapp/fc/issues/448
- metrics：https://github.com/devsapp/fc/issues/447
- nas：https://github.com/devsapp/fc/issues/446
- ondemand：https://github.com/devsapp/fc/issues/445
- provision ：https://github.com/devsapp/fc/issues/444
- proxied：https://github.com/devsapp/fc/issues/443
- remote：https://github.com/devsapp/fc/issues/442
- remove：https://github.com/devsapp/fc/issues/441
- stress：https://github.com/devsapp/fc/issues/440
- sync：https://github.com/devsapp/fc/issues/439
- version：https://github.com/devsapp/fc/issues/438
- tips：https://github.com/devsapp/fc/issues/437
- invoke：https://github.com/devsapp/fc/issues/436
- info：https://github.com/devsapp/fc/issues/435
- Fun2s： https://github.com/devsapp/fc/issues/434
- deploy：https://github.com/devsapp/fc/issues/432
- build：https://github.com/devsapp/fc/issues/431
- alias：https://github.com/devsapp/fc/issues/430

- [x] eval 文档完善：https://github.com/devsapp/fc/issues/417
- [x] s stress相关命令文档需要补充完整：https://github.com/devsapp/fc/issues/422
- [ ] 本地调试文档补充：https://github.com/devsapp/fc/issues/424
- [x] 端云联调：权限相关说明：https://github.com/devsapp/fc/issues/100
